tests/kernel_bundle/has_kernel_bundle_zero_devices.h

```diff
@@ -0,0 +1,110 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides common code for zero devices test case
```

Comment:
I am puzzled by the plural of devices everywhere when there is 0 device.
Even the files names are about "devices"...

Reply:
"zero devices" was replaced with "zero device" in this commit: f27ce9f11df823b5b2a9c41db4d27fb0ac20b981